### ISSUE TITLE
UDP communication with ESP devices

### ISSUE THREAD

**Issue by mistergreen** [mistergreen/WAAC#7]
I was thinking the ESP web server could tell another ESP to turn on and off or fade PWM.
I think UDP is best for local wifi communication.

**Comment by LuckyLuzz** [comment]
yeah that would be nice, have a look at (http://sonoff.itead.cc/en/)   You can buy that stuff very cheap and it is running on ESP8266. I guess most people connect those divces over a raspberry pi running node red  (https://nodered.org/)

How about a function were the controller can sent informations to "Siri" or another speaking device?

**Comment by mistergreen** [comment]
The sonoff devices runs on MQTT protocol. It would talk a long while to figure out how everything would work. Node Red is awesome.

What function would sending to 'siri' do?

**Comment by LuckyLuzz** [comment]
The "Alexa or Siri" function would connect the ESP to Alexa or Siri. So you could ask your Alexa: "Alexa how are my fish doing?" She would get the variables for ph and temp and answer: "You Fish doing good, the water is warm and the water is good"     something like that. There are a couple totorials to connect the Esp to Alexa and turn on/off lights an stuff. So the aquarium controller comminicating with Alexa would be possible I guess.

**Comment by mistergreen** [comment]
Ah, I see. It is possible an example here
https://create.arduino.cc/projecthub/adithya-tg/alexa-based-smart-home-monitoring-e36b7f?ref=tag&ref_id=alexa&offset=3

You'd have to set up a few online API and maybe a local server on a cheap computer or a raspberry Pi.
